packages/destructure/dev/index.tsx

```diff
@@ -43,7 +43,7 @@ const App: Component = () => {
       <div class="wrapper-v" onclick={() => setCount(p => ++p)}>
         <button class="btn">{count()}</button>
       </div>
-      <Inner seconds={seconds()} count={count()} list={list()} />
+      {/* <Inner seconds={seconds()} count={count()} list={list()} /> */}
```

Comment:
Why that change?

Reply:
I honesty don't remember, for the destructure I was treating `dev/` as a playground than a demo of how things work.
Not that it should have been committed either way...